src/OpenAPM.ts

```diff
@@ -165,20 +165,6 @@ export class OpenAPM extends LevitateEvents {
     );
   };
 
-  private gracefullyShutdownMetricsServer = () => {
-    this.metricsServer?.close((err) => {
-      promClient.register.clear();
-      console.log('Shutting down metrics server.');
-      if (err) {
-        console.error('Error while shutting down the metrics server:', err);
-        process.exit(1);
```

Comment:
@chtushar This task is misunderstood. You need to clear PromClient register and call the `this.metricsServer.close()` but don't want to exit the process. You have removed the code completely

![image](https://github.com/last9/openapm-nodejs/assets/621238/bc823f86-10eb-4eda-a276-207ac172bd94)